jbmc/src/java_bytecode/java_entry_point.cpp

```diff
@@ -65,6 +65,30 @@ static bool should_init_symbol(const symbolt &sym)
   return is_java_string_literal_id(sym.name);
 }
 
+/// If symbol is a class literal, and an appropriate initializer method exists,
+/// return a pointer to its symbol. If not, return null.
+/// \param symbol: possible class literal symbol
+/// \param symbol_table: table to search
+/// \return pointer to the initializer method symbol or null
+static const symbolt *get_class_literal_initializer(
+  const symbolt &symbol,
+  const symbol_table_baset &symbol_table)
+{
+  if(symbol.value.is_not_nil())
+    return nullptr;
+  if(symbol.type != symbol_typet("java::java.lang.Class"))
+    return nullptr;
+  if(!has_suffix(id2string(symbol.name), "@class_model"))
+    return nullptr;
+  return symbol_table.lookup(
+    "java::java.lang.Class.cproverInitializeClassLiteral:(ZZZZZZZ)V");
```

Comment:
What is with the `ZZZZZZZ`?

Reply:
Type descriptor -- 7 boolean parameters, void return.